frontend/doc-site/04__Analyze Public Data/4_4__Annotate.mdx

```diff
@@ -0,0 +1,182 @@
+
+
+#### Prediction model
+You can download models to perform prediction for the following tissues:
+
+For use in `venv` environments:
+- [HLCA Lung Model](https://cellxgene-annotation-public.s3.us-west-2.amazonaws.com/cell_type/models/hlca_20220728144320_venv.zip)
+- [Pan Tissue Immune Reference](https://cellxgene-annotation-public.s3.us-west-2.amazonaws.com/cell_type/models/immune_ref_20220728145305_venv.zip)
+
+For use in `conda` environments:
+- [HLCA Lung Model](https://cellxgene-annotation-public.s3.us-west-2.amazonaws.com/cell_type/models/hlca_20220728144320_conda.zip)
+- [Pan Tissue Immune Reference](https://cellxgene-annotation-public.s3.us-west-2.amazonaws.com/cell_type/models/immune_ref_20220728145305_conda.zip)
+
+
+
+### Step 1: Running `cellxgene annotate`
+
+After installing the latest version of CELLxGENE and the annotate subcommand, you can run a standard call to the command like so (see `cellxgene annotate --help` for a full list of available options):
+
+`cellxgene annotate  --input-h5ad-file ./pathTo/data.h5ad --model-url modelURL --output-h5ad-file ./pathTo/annotatedData.h5ad`
+
+In the call of the `annotate` subcommand above, we specify three arguments
+- the local path to your query dataset (in `anndata` format)
+- the url of the tissue model which you like to use to perform reference mapping (for a list of available reference models, refer to the [section above](#prediction-model))
+- the local path where `cellxgene annotate` will write a new `anndata` object that includes the predicted annotations
+
+**Note:** the first time use of a given model will:
+- trigger a one-time download of the model, stored locally for repeated use (in .model_cache by default)
+- take some extra time to setup an (internal) python env in which to run the model
+
+Once your call to `cellxgene annotate` completes, you will have the following new information in your `anndata` object:
+
+- transferred annotations/labels [`categorical`] (i.e. cell type) - these will be stored under `adata.obs['cxg_cell_type_predicted']` by default
+- confidence scores [`numeric`] - stored under `adata.obs['cxg_cell_type_predicted_uncertainty']` by default
+- a mapping into the reference embedding space - stored under `adata.obsm['cxg_cell_type'], by default`
+- a UMAP layout for the query data in this reference embedding space - stored under `adata.obsm['cxg_cell_type_umap']` by default
+- metadata about the model used - stored under `adata.uns['cxg_cell_type_predictions']['model_url']` by default
+
+The `cellxgene annotate` subcommand offers a number of options to customize how the workflow is run and to adapt to different scenarios. You can display the full list of available options like so:
+
+`cellxgene annotate -h`
+
+
+### Step 2: Explore and Refine
+
+After you have successfully completed the annotation workflow, you are now ready to explore your results in CELLxGENE Desktop!
```

Comment:
```suggestion
After you have successfully completed the annotation workflow, you are now ready to explore your results in CELLxGENE Annotate!
```

Reply:
Wondering if we should keep it as desktop since we are still using the desktop explorer to view the data/annotations?

Reply:
I thought the rebranding renamed "cellxgene desktop" to "CZ CELLxGENE Annotate"

Reply:
You are right - my understanding was not up-to-date, I'll commit all of these relevant changes that you have suggested here - thanks!